lib/api/multiObjectDelete.js

```diff
@@ -166,6 +169,60 @@ function _parseXml(xmlToParse, next) {
     });
 }
 
+/**
+ * processObjectVersion - process object version to be deleted
+ * @param {object} entry - entry from data model
+ * @param {string} bucketName - bucket name
+ * @param {function} next - callback to call with error or decoded version
+ * @return {undefined}
+ **/
+function processObjectVersion(entry, bucketName) {
```

Comment:
not clear what this does (and it certainly does not "process" a version): and especially when reading in the waterflow, extracting this function completely hides what is done...

maybe it should just be renamed to `decodeObjectVersion` ?

Reply:
Done: https://github.com/scality/cloudserver/pull/5195/commits/21d22ad1832687ed188f284dcafeaf6ae1c4f285#diff-e8319ec288d282012f389d259d6468a0922b91ec7cc52f2b8c15b01ff3ba9c2aR178